vmm/src/vm.rs

```diff
@@ -31,7 +31,7 @@ use kvm_bindings::{
 };
 use kvm_ioctls::*;
 use libc::O_TMPFILE;
-use libc::{c_void, siginfo_t, EFD_NONBLOCK};
+use libc::{c_int, c_void, siginfo_t, EFD_NONBLOCK, SA_SIGINFO, SIGHUP, SIGINT, SIGWINCH};
```

Comment:
Based on comment below, please add `TIOCGWINSZ` to the list.

Reply:
Done.